src/loggerBuilder.ts

```diff
@@ -0,0 +1,52 @@
+import { createLogger, format } from 'winston';
+import Transport from 'winston-transport';
+
+const { combine, splat, timestamp, printf } = format;
+
+const myFormat = printf(info => {
+    let msg = `${info.message} ${info.timestamp} `;
+    if (info.meta) {
+        msg += JSON.stringify(info.meta);
+    }
+    return msg;
```

Comment:
What is an example of this log message?

Reply:
Good question. Will add an example screenshot from CloudWatch.